tests/unit/pocket-relayer.unit.ts

```diff
@@ -275,12 +266,13 @@ describe('Pocket relayer service (unit)', () => {
   })
 
   it('throws an error when loading an invalid blockchain', async () => {
+    axiosMock.reset()
```

Comment:
Why is this needed here? Doesn't this happen after each test case already with the added code?

Reply:
yes and that's exactly why I put it there, to remove the blockchain response so it returns 404 and mimics the not found test